alibi_detect/saving/loading.py

```diff
@@ -28,60 +28,38 @@
 
 logger = logging.getLogger(__name__)
 
-REQUIRES_BACKEND = [
-    'ClassifierDrift',
-    'ClassifierUncertaintyDrift',
-    'LearnedKernelDrift',
-    'LSDDDrift',
-    'MMDDrift',
-    'RegressorUncertaintyDrift',
-    'SpotTheDiffDrift'
-]
```

Comment:
Why is this no longer necessary?

Reply:
`_init_config()` has been simplified by the use of `from_config()`, and we no longer need to manually update `backend`:

https://github.com/SeldonIO/alibi-detect/blob/644ec298f73674175e605696ceeab5859864198a/alibi_detect/saving/loading.py#L160-L166